examples/graceful_shutdown.py

```diff
@@ -39,8 +39,8 @@ async def worker(app):
             }
             fs.append(stream.send_json(data, id=now.timestamp()))
 
-        # Run in parallel
-        await asyncio.gather(*fs)
+        # Run in parallel and don't fail even on delivery exception
+        await asyncio.gather(*fs, return_exceptions=True)
```

Comment:
I think there's some concern here that you could ignore other exceptions. I wonder if it makes more sense to gracefully handle the exception within the tasks..

Reply:
As far as I understand, the meaning of this file is to show an example of working with graceful shutdown and reconnection from the moment of the break.

But during testing, I closed the tab, which led to the silent crash of this worker. As a result, the example no longer worked in a new open tab.

A worker should not fall due to failure to complete tasks. But in this context, I would not add a lot of code to process such events - this may confuse an inexperienced reader.

Reply:
Yes, I agree with your concern, I'm just suggesting that there is probably a better way to achieve it.
If we want to ensure this keeps running on any exception, then there should probably an explicit log call for any returned exceptions.

Reply:
wrapped tasks and added print with event id and exception